build/package/scripts/build-gradle.sh

```diff
@@ -63,33 +64,43 @@ echo
 export ODS_OUTPUT_DIR=${OUTPUT_DIR}
 echo "Exported env var 'ODS_OUTPUT_DIR' with value '${OUTPUT_DIR}'"
 echo
-echo "Building (Compile and Test) ..."
-# shellcheck disable=SC2086
-./gradlew clean build ${GRADLE_ADDITIONAL_TASKS} ${GRADLE_OPTIONS}
-echo
 
-echo "Verifying unit test report was generated  ..."
-BUILD_DIR="build"
-UNIT_TEST_RESULT_DIR="${BUILD_DIR}/test-results/test"
+echo "Building ..."
+UNIT_TEST_ARTIFACTS_DIR="${ARTIFACTS_DIR}/xunit-reports"
+CODE_COVERAGE_ARTIFACTS_DIR="${ARTIFACTS_DIR}/code-coverage"
 
-if [ -d "${UNIT_TEST_RESULT_DIR}" ]; then
-    UNIT_TEST_ARTIFACTS_DIR="${ARTIFACTS_DIR}/xunit-reports"
-    mkdir -p "${UNIT_TEST_ARTIFACTS_DIR}"
-    cp "${UNIT_TEST_RESULT_DIR}/"*.xml "${UNIT_TEST_ARTIFACTS_DIR}/${ARTIFACT_PREFIX}"
-else
-  echo "Build failed: no unit test results found in ${UNIT_TEST_RESULT_DIR}"
-  exit 1
-fi
+if [ -f "${UNIT_TEST_ARTIFACTS_DIR}/${ARTIFACT_PREFIX}report.xml" ] && [ -f "${CODE_COVERAGE_ARTIFACTS_DIR}/${ARTIFACT_PREFIX}coverage.xml" ]; then
```

Comment:
Do we just have one report XML actually? I remember a discussion with @oalyman about multiple JUnit files being generated ...

Reply:
you are right, I'll work on it

Reply:
see https://github.com/opendevstack/ods-pipeline/commit/729986c3c3d346308b9f24af985f312e6ef9eb3a